### DISCUSSION TITLE
power issues

### DISCUSSION THREAD

**Discussion by blu31c33** [tesla-android/issue-tracker#24] [Q&A]
I have a 2019 Tesla m3 and cannot get stable power across both pi's.  my center console has the older usb a ports, not usb c.  I tried using a 12v power adapter (with usb a to usb c cable) and still am running into issues.

what's a stable configuration?  do I need a 12v usb c adapter instead of the usb a to usb c?  that's the only thing I can think.  I've tried using the front two usb, one front and one rear, and a basenor usb hub (adds to usb c ports to front along with usb a ports).

thanks!

**Comment by galosre** [comment]
Even on my Y 2022 the USB-C power is kind of weak, I am using an adapter in the cigarette lighter

**Reply by mikegapinski** [reply]
Power consumption and heat output will be greatly reduced in the next version. It’s already fixed, I am also fixing other stuff before I release an update
